src/ClientHandle.h

```diff
@@ -377,13 +385,18 @@ class cClientHandle  // tolua_export
 
 	bool IsPlayerChunkSent();
 
+	cForgeHandshake m_ForgeHandshake;
+
+	std::unique_ptr<cForgeMods> m_ForgeMods;
+
```

Comment:
Moved to private, added public accessors for cProtocols. Is there a preference for the ordering of data members? Noticed it seems to be a more logical grouping than alphabetical, and the order has to be matched by the constructor initializer list or compiler error. For now I put m_Forge\* at the top